sites/docs/src/content/docs/usage/configuration/introduction.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Configuration
+subtitle: How configure nf-core pipelines
+shortTitle: Configuration options
+weight: 1
+parentWeight: 20
+---
+
+## Configure nf-core pipelines
+
+Each nf-core pipeline comes with a set of “sensible defaults”.
+While the defaults are a great place to start, you will certainly want to modify these to fit your own purposes and system requirements.
+
+When a pipeline is launched, Nextflow will look for config files in several locations.
+As each source can contain conflicting settings, the sources are ranked to decide which settings to apply.
+
+nf-core pipelines may utilize any of these configuration files.
+
+Configuration sources are reported below and listed in order of priority:
+
+1. Parameters specified on the command line (`--parameter`)
```

Comment:
I think we should drop this list to rather point towards the Nextflow docs, unless this changes

Reply:
I see this as minimal framing for the rest of the section. E.g., how nextflow.config and parameter files fit together.